python/dgl/graph.py

```diff
@@ -1814,6 +1842,7 @@ def from_scipy_sparse_matrix(self, spmat, multigraph=False):
             The graph's adjacency matrix
 
         multigraph : bool, optional
+            Deprecated (Will be deleted in the future).
```

Comment:
Change the code accordingly? The `multigraph` argument should be deleted for `graph_index.from_scipy_sparse_matrix`.

Reply:
I think we still need to keep the top level API the same and print a deprecated message